build-parent/pom.xml

```diff
@@ -53,6 +53,11 @@
         <!-- Enable APT by default for Eclipse -->
         <m2e.apt.activation>jdt_apt</m2e.apt.activation>
 
+        <!-- These 2 properties are used in catalog-overrides.json -->
+        <!-- Supported Java versions, interpreted as a version range -->
+        <supported-java-versions>[${maven.compiler.release},)</supported-java-versions>
```

Comment:
@gastaldi what does this generate? `[...,)`

Reply:
`[11,)` meaning any version >=11

Reply:
We probably don't need a complete range expression support for Java versions. Could we simply have something like `min-java-version`? Would we also need `max-java-version`?

Reply:
I don't think it makes sense to specify a `max-java-version` because in theory, you can always use the latest and greatest JDK release. 

`min-java-version` makes sense and it should match the `${maven.compiler.release}`

Reply:
@ia3andy WDYT?

Reply:
What about having just `minimum-java-version` and `recommended-java-version`?

Reply:
> What about having just `minimum-java-version` and `recommended-java-version`?

That sounds good

Reply:
Done!